ws/templates/help/leaders/ws_gear.html

```diff
@@ -77,8 +77,13 @@ <h4>B Trip</h4>
   <ul>
     <li>All the group gear required for an A Trip +</li>
     <li>Two sleeping bags (one winter (15F) rated and one three-season (35F) rated minimum)</li>
-    <li>One bivy or bothy bag - A large MITOC tarp, an actual bothy bag, or a bivy bag such as the OR Alpine Bivy)</li>
-    <li>One shelter - A three-season or four-season tent and if weather conditions permit a bothy or tarp</li>
+    <li>One emergency shelter for 2 people, ex:</li>
```

Comment:
```suggestion
    <li>Emergency shelter for 2 people. For example, any of:</li>
```

(Only to be extremely clear that a single bivy does not count)